modules/planning/tasks/deciders/path_assessment_decider/path_assessment_decider.cc

```diff
@@ -217,29 +218,58 @@ Status PathAssessmentDecider::Process(
   }
   reference_line_info->SetCandidatePathData(new_candidate_path_data);
 
+  // 4. Update necessary info for lane-borrow decider's future uses.
+  // Update front static obstacle's info.
   if (!(reference_line_info->GetBlockingObstacleId()).empty()) {
-    if (PlanningContext::Instance()->front_static_obstacle_cycle_counter() <
-        0) {
-      PlanningContext::Instance()->ResetFrontStaticObstacleCycleCounter();
+    if (PlanningContext::Instance()->path_decider_info().
```

Comment:
move PathDeciderInfo into PlanningStatus?
can be done later though

Reply:
Got it. Later I will move it into PlanningStatus, and rename it to be PathDeciderStatus.